build_runner/README.md

```diff
@@ -99,8 +99,9 @@ Some commands also have additional options:
 ##### serve
 
 - `--hostname`: The host to run the server on.
-- `--hot-reload`: Enables automatic module reloading on rebuilds. 
-See [hot-module-reloading](../docs/hot_module_reloading.md) for more info.
+- `--live-reload`: Enables automatic page reloading on rebuilds.
```

Comment:
If you provide both which one takes precedence? Could potentially model this as an enum to prevent that issue.